public/app/features/transformers/docs/content.ts

```diff
@@ -628,6 +628,68 @@ Use this transformation to construct a matrix by specifying fields from your que
+  )}
+
+  Once \`Calculate\` has been selected, another select box will appear next to the respsective field which will allow statistics to be selected:
+
+  ${buildImageContent(
+    '/static/img/docs/transformations/nested-table-select-stat.png',
+    imageRenderType,
+    'A select box showing available statistic calculations once the calculate option for the field has been selected.'
+  )}
+
+  For information about available calculations, refer to [Calculation types][].
+
+  Here's an example of original data:
+
+  | Time                | Server ID | CPU Temperature | Server Status |
+  | ------------------- | --------- | --------------- | ------------- |
+  | 2020-07-07 11:34:20 | server 1  | 80              | Shutdown      |
+  | 2020-07-07 11:34:20 | server 3  | 62              | OK            |
+  | 2020-07-07 10:32:20 | server 2  | 90              | Overload      |
+  | 2020-07-07 10:31:22 | server 3  | 55              | OK            |
+  | 2020-07-07 09:30:57 | server 3  | 62              | Rebooting     |
+  | 2020-07-07 09:30:05 | server 2  | 88              | OK            |
+  | 2020-07-07 09:28:06 | server 1  | 80              | OK            |
+  | 2020-07-07 09:25:05 | server 2  | 88              | OK            |
+  | 2020-07-07 09:23:07 | server 1  | 86              | OK            |
+
+  This transformation has two steps. First, specify one or more fields by which to group the data. This groups all the same values of those fields together, as if you sorted them. For instance, if you group by the Server ID field, Grafana groups the data this way:
+
+  | Server ID      |  |
+  | -------------- | ------------- |
+  | server 1 | <table><th><tr><td>Time</td><td>CPU Temperature</td><td>Server Status</td></tr></th><tbody><tr><td>2020-07-07 11:34:20</td><td>80</td><td>Shutdown</td></tr><tr><td>2020-07-07 09:28:06</td><td>80</td><td>OK</td></tr><tr><td>2020-07-07 09:23:07</td><td>86</td><td>OK</td></tr></tbody></table> |
+  | server 2 | <table><th><tr><td>Time</td><td>CPU Temperature</td><td>Server Status</td></tr></th><tbody><tr><td>2020-07-07 10:32:20</td><td>90</td><td>Overload</td></tr><tr><td>2020-07-07 09:30:05</td><td>88</td><td>OK</td></tr><tr><td>2020-07-07 09:25:05</td><td>88</td><td>OK</td></tr></tbody></table> |
+  | server 3 | <table><th><tr><td>Time</td><td>CPU Temperature</td><td>Server Status</td></tr></th><tbody><tr><td>2020-07-07 11:34:20</td><td>62</td><td>OK</td></tr><tr><td>2020-07-07 10:31:22</td><td>55</td><td>OK</td></tr><tr><td>2020-07-07 09:30:57</td><td>62</td><td>Rebooting</td></tr></tbody></table> |
+
+  After choosing the field by which you want to group your data, you can add various calculations on the other fields and apply the calculation to each group of rows. For instance, you might want to calculate the average CPU temperature for each of those servers. To do so, add the \`mean\` calculation applied on the CPU Temperature field to get the following result:
```

Comment:
For UI elements, we bold them in the docs. I'm not sure if this "mean" that you're referring to is something that can be selected in the UI. If so, it should be in bold.

Reply:
That makes sense, I'll do that from now on 😄